### PR TITLE
gitflow finish command can publish to remote if needed

### PR DESCRIPTION
**Pull request by dixiedream**
Yeah it's me @aleDVirgo with my personal github account, here is the PR as requested... sorry I'm not a Golang programmer but I've experimented with it.

The goal of this PR is to been able to publish/push the git flow finish command output also on remote cuz it's a common practice here in the place where I work.

The new `willPush` boolean var can be triggered by a conf var or maybe by asking the user in a prompt after the finish command has been selected.

Thanks, hoping to not have written some bad s**t

### CODE REVIEW

File: pkg/commands/git_commands/flow.go [hunk 1]
```diff
@@ -48,6 +48,10 @@ func (self *FlowCommands) FinishCmdObj(branchName string) (oscommands.ICmdObj, e
 		return nil, errors.New(self.Tr.NotAGitFlowBranch)
 	}
 
+	if willPush == true {
```
[the remark this answers is not in the document]
Comment (dixiedream) [reply pkg/commands/git_commands/flow.go:51]: Or to be more precise `self.UserConfig.Git.GitFlowFlags.PublishChanges`

[the remark this answers is not in the document]
Comment (mark2185) [reply pkg/commands/git_commands/flow.go:51]: My bad, I meant to write:
```
if flags := self.UserConfig.Git.GitFlowFlags; flags != "" {
    return self.cmd.New("git flow " + branchType + " finish " + flags + " + suffix), nil
}
```

[the remark this answers is not in the document]
Comment (dixiedream) [reply pkg/commands/git_commands/flow.go:51]: Yeah it's fine, even so I think your solution it's a bit more "power user" driven than mine cause you need to know what flags are and what they do but in the end it'll get the work done.

**Comment by mark2185** [comment]
@dixiedream was this closed on purpose?

**Pull request closed without merging**